spring-boot-docs/src/main/asciidoc/using-spring-boot.adoc

```diff
@@ -824,7 +824,7 @@ This allows you to attach a debugger to your packaged application:
 === Using the Maven plugin
 The Spring Boot Maven plugin includes a `run` goal which can be used to quickly compile
 and run your application. Applications run in an exploded form, and you can edit
-resources for instant "`hot`" reload.
+resources for instant "`hot`" reload, when the fork mode is enabled in the plugin.
```

Comment:
I think this was referring to the `addResources` feature and not really devtools but granted we should probably rephrase this.